react/AddToCartButton.tsx

```diff
@@ -193,7 +197,7 @@ function AddToCartButton(props: Props) {
       return
     }
 
-    addItem(skuItems, { ...utmParams, ...utmiParams })
+    addItem(skuItems, { ...utmParams, ...utmiParams }, undefined, options)
```

Comment:
@lucasecdb help me here. What are the odds of sending an undefined `salesChannel`?
It was already undefined before @silviasfon's change but it seems an opportunity to improve it.

Reply:
I suggested another way to implement this change [here](https://github.com/vtex-apps/order-items/pull/41#discussion_r611863882) which should fix this issue 😄